cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -615,24 +585,14 @@ setNodeThread lvbe nodeThr =
   modifyMVar_ (getbe lvbe) $ \lvs ->
       return $ lvs { lvsNodeThread = LiveViewThread $ Just nodeThr }
 
-setNodeKernel :: (NFData a, HasTxMaxSize (ExtLedgerState blk))
-              => LiveViewBackend blk a
-              -> NodeKernel IO RemoteConnectionId LocalConnectionId blk
-              -> IO ()
-setNodeKernel lvbe nodeKern =
-    modifyMVar_ (getbe lvbe) $ \lvs -> do
-        -- This is correct for Byron and Shelley.
-        MempoolCapacityBytes mempoolCapacity <- STM.atomically $ getCapacity (getMempool nodeKern)
-
-        currentLedger <- STM.atomically $ getCurrentLedger (getChainDB nodeKern)
-
-        pure lvs
-              { lvsNodeKernel = SJust (LVNodeKernel nodeKern)
-              , lvsMempoolCapacity =
-                  fromIntegral $
-                    mempoolCapacity `div` getMaxTxSize currentLedger
-              , lvsMempoolCapacityBytes = fromIntegral mempoolCapacity
-              }
+storePeersInLiveView :: NFData a => [LVPeer blk] -> LiveViewBackend blk a -> IO ()
+storePeersInLiveView peers lvbe = modifyMVar_ (getbe lvbe) $ \lvs -> return $ lvs { lvsPeers = peers }
+
+storeMempoolDataInLiveView :: NFData a => Word64 -> Word64 -> LiveViewBackend blk a -> IO ()
+storeMempoolDataInLiveView capacity capacityBytes lvbe = modifyMVar_ (getbe lvbe) $ \lvs ->
+  return $ lvs { lvsMempoolCapacity = capacity
+               , lvsMempoolCapacityBytes = capacityBytes
+               } 
```

Comment:
This reads less well than:
```
storeMempoolDataInLiveView capacity capacityBytes lvbe =
  modifyMVar_ (getbe lvbe) $ \lvs ->
    return $ lvs { lvsMempoolCapacity = capacity
                 , lvsMempoolCapacityBytes = capacityBytes
                 } 
```
Also `return` is going to be replaced with `pure` and `pure` is a better name for what it does. Maybe we should start using `pure` instead.

Reply:
Done.